package.json

```
@@ -10,22 +10,27 @@
     "nodejs",
     "micro-services"
   ],
+  "engine": {
+    "node": ">=16.0"
+  },
```

nitpick, but this needs to be `engines` to work IMO

```suggestion
  "engines": {
    "node": ">=16.0"
  },
```